src/server/db/reporting-periods.js

```diff
@@ -146,25 +155,31 @@ async function closeReportingPeriod (user, period, trns = knex) {
     .where({ id: reporting_period_id })
     .update({
       certified_at: new Date().toISOString(),
+      // TODO(mbroussard): is this correct? We write a stringified json of user row?
```

Comment:
That does seem weird.  I can open a ticket and look into it separately.

Reply:
#332 removed this.